Vagrantfile

```diff
@@ -46,7 +53,7 @@ Vagrant.configure(2) do |config|
   config.vm.hostname = options['machine_name']
 
   # network settings
-  config.vm.network 'private_network', ip: options['ip']
+  config.vm.network 'private_network', ip: options['netmask']+'.'+options['ip']
```

Comment:
what does this do? A netmask is usually of the form `255.255.255.0` or similar, it does not make sense to concat these.